scripts/recipes/bread_blocks.zs

```diff
@@ -0,0 +1,13 @@
+print ("Patching bread block recipes");
+
+//start with all wheat bread items. (population: 1)
+val tagBread = <tag:items:c:bread/wheat>;
+
+//remove original recipe
+craftingTable.removeByName("architects_palette:bread_block");
+
+//add storage block recipes
+craftingTable.addShapeless("craft_bread_block", <item:architects_palette:bread_block> * 1, [tagBread, tagBread, tagBread, tagBread, tagBread, tagBread, tagBread, tagBread, tagBread]);
+craftingTable.addShapeless("uncraft_bread_block", <item:minecraft:bread> * 9, [<item:architects_palette:bread_block>]);

```

Comment:
convention is to prefix our recipes with `hexxycraft`, at least the ones from crafttweaker
```suggestion
craftingTable.addShapeless("hexxycraft/bread_block/craft", <item:architects_palette:bread_block> * 1, [tagBread, tagBread, tagBread, tagBread, tagBread, tagBread, tagBread, tagBread, tagBread]);
craftingTable.addShapeless("hexxycraft/bread_block/uncraft", <item:minecraft:bread> * 9, [<item:architects_palette:bread_block>]);

```

Reply:
Good point. TBH I could've done better with those names. 
That said, shouldn't it be `hexxycraft:`, So it's its own namespace instead of part of the `minecraft:` namespace?
Answer that, then I'll commit whatever it is you decide on.

Reply:
I think it's actually in the crafttweaker namespace. I tried `hexxycraft:` at one point and it didn't work.

Reply:
Okey-doke-lokey.